docker/dev/django/container-start.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+python manage.py makemigrations && \
```

Comment:
What is this change for?

Reply:
Whenever a change is made to django make we have to run the command makemigrations separate every time. Shouldn't we put it in the start so all migrations are made and then migrated to db?